enterprise/backend/test/metabase_enterprise/audit_app/pages_test.clj

```diff
@@ -224,3 +224,15 @@
                       (sort-by first)
                       (filter #((set (map u/the-id [a b c d e])) (first %)))
                       (map #(nth % 6))))))))))
+
+(deftest active-and-new-by-time-test
+  (testing "The active-and-new-by-time audit query correctly returns results (#32244)"
+    (mt/with-test-user :crowberto
+      (premium-features-test/with-premium-features #{:audit-app}
+        (let [results (mt/user-http-request :crowberto :post 202 "dataset"
+                                            {:type :internal
+                                             :fn   "metabase-enterprise.audit-app.pages.users/active-and-new-by-time"
+                                             :args ["day"]})]
+          (def results results)
```

Comment:
```suggestion
```

Reply:
thanks for catching, very weird that this didn't show up in CI